homeassistant/components/mqtt/client.py

```diff
@@ -77,7 +77,6 @@
 )
 from .models import (
     DATA_MQTT,
-    AsyncMessageCallbackType,
```

Comment:
Seems we do not use this alias anywhere else

Reply:
Do you want me to remove it or should we keep it around?

Reply:
Good question, as it is not exported, it seems we could clean it up here.

Reply:
Thnx!